xml/ntp_banners.xml

```diff
@@ -324,13 +324,17 @@
       version="ntpd 4.2.6p2@1.2194 Wed Nov 24 15:54:11 UTC 2010 (1)",
       processor="i386", system="FreeBSD/6.3-NETSCALER-9.3", leap=00, stratum=3,
     </example>
-    <param pos="0" name="service.family" value="NTP"/>
-    <param pos="0" name="service.product" value="NTP"/>
     <param pos="1" name="service.version"/>
-    <param pos="0" name="os.vendor" value="Citrix"/>
-    <param pos="0" name="os.product" value="NetScaler"/>
+    <param pos="0" name="service.vendor" value="Citrix"/>
+    <param pos="0" name="service.family" value="NetScaler"/>
+    <param pos="0" name="service.product" value="NetScaler"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:citrix:netscaler:{service.version}"/>
+    <param pos="0" name="os.vendor" value="FreeBSD"/>
```

Comment:
Its not really FreeBSD (they customize quite a bit). Can we either move this back to NetScaler or add a hw.* assert for NetScaler? os.family as FreeBSD makes sense.

Reply:
Fixed.